docs/pages/management/export-audit-events/splunk.mdx

```diff
@@ -0,0 +1,783 @@
+respectively). Later, you will configure the plugin to use these credentials to
+connect to the Auth Service.
+
+</TabItem>
+</Tabs>
+
+</ScopedBlock>
+
+<ScopedBlock scope="cloud">
+
+The following `tctl auth sign` command impersonates the `teleport-event-handler`
+user, generates signed credentials, and writes an identity file to the local
+directory:
+
+```code
+$ tctl auth sign --user=teleport-event-handler --out=auth.pem
+```
+
+Teleport's Event Handler plugin listens for new and updated audit logs by
+connecting to the Teleport Auth Service's gRPC endpoint over TLS.
+
+The identity file, `auth.pem`, includes both TLS and SSH credentials. The Event
+Handler plugin uses the SSH credentials to connect to the Proxy Service, which
+establishes a reverse tunnel connection to the Auth Service. The plugin uses
+this reverse tunnel, along with your TLS credentials, to connect to the Auth
+Service's gRPC endpoint.
+
+You will refer to this file later when configuring the plugin.
+
+</ScopedBlock>
+
+Move the credentials you generated to the host where you are running the Event
+Handler plugin.
+
+<Admonition
+  title="Certificate Lifetime"
+>
+
+  By default, `tctl auth sign` produces certificates with a relatively short
+  lifetime. For production deployments, you can use the `--ttl` flag to ensure a
+  more practical certificate lifetime, e.g., `--ttl=8760h` to export a one-year
+  certificate.
+
+</Admonition>
+
+## Step 2/4. Configure the Universal Forwarder
+
+In this step, you will configure the Universal Forwarder to receive audit logs
+from the Teleport Event Handler plugin and forward them to Splunk. The Event
+Handler sends audit logs as HTTP POST requests with the content type
+`application/json`.
+
+We will assume that you assigned `$SPLUNK_HOME` to `/opt/splunkforwarder` when
+installing the Universal Forwarder. 
+
+<Details title="Finding your $SPLUNK_HOME">
+
+To find your `$SPLUNK_HOME`, run the following command to see the location your
+Universal Forwarder service definition, which the init system systemd uses to
+run the Universal Forwarder:
+
+```code
+$ sudo systemctl status SplunkForwarder.service
+● SplunkForwarder.service - Systemd service file for Splunk, generated by 'splunk enable boot-start'
+     Loaded: loaded (/lib/systemd/system/SplunkForwarder.service; enabled; vendor preset: enabled)
+     Active: active (running) since Fri 2022-10-07 15:57:37 UTC; 2h 18min ago
+   Main PID: 1772 (splunkd)
+      Tasks: 53 (limit: 2309)
+     Memory: 70.8M (limit: 1.8G)
+     CGroup: /system.slice/SplunkForwarder.service
+             ├─1772 splunkd --under-systemd --systemd-delegate=yes -p 8089 _internal_launch_under_systemd
+             └─1810 [splunkd pid=1772] splunkd --under-systemd --systemd-delegate=yes -p 8089 _internal_launch_under_systemd [process-runner]
+```
+
+View the file at the path shown in the `Loaded:` field. Your `$SPLUNK_HOME`
+will include the filepath segments in `ExecStart` before `bin`.
+
```

Comment:
Just a suggestion: We could include an example `cat /lib/systemd/system/SplunkForwarder.service` and show the relevant output here.